### PR TITLE
set-ios-product-bundle-identifier 1.0.0

### PR DESCRIPTION
**Pull request by emenegro**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/set-ios-product-bundle-identifier/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- script
+- build
+- utils
+- xcode
+- xamarin
```
Comment (viktorbenei) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: type tags require a bit of normalization - @godrei can you please review & advise?

**Comment by viktorbenei** [comment]
Everything else is :+1: , thanks @emenegro !

@godrei pls review the tags asap, and do your review.

My review: everything's OK except the type tags.

File: steps/set-ios-product-bundle-identifier/1.0.0/step.yml (hunk 1 above)
Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: Sorry, I have committed leaving only three.

Comment (viktorbenei) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: No problem, I'd wait for @godrei to advise as we're kinda in the middle of a type tag revision, for better discoverability in the workflow editor etc

File: steps/set-ios-product-bundle-identifier/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+support_url: https://github.com/emenegro/bitrise-step-product-bundle-identifier/issues
+published_at: 2017-03-28T11:05:05.50225827+02:00
+source:
+  git: https://github.com/emenegro/bitrise-step-product-bundle-identifier.git
+  commit: f218dde9c751806bc952183d682005d072012207
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
```
Comment (trapacska) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: Could you please change `type_tags` from the following available options:
type_tags:
- access control
- artifact info
- installer
- deploy
- utility
- dependency
- code sign
- build
- test
- notification

Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: Are these ok?

```
- build
- utility
```

Thanks.

Comment (godrei) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: i think it should be `utility` only, since, the step does not do any build.

Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.0.0/step.yml]: Ok!

**Comment by godrei** [comment]
@emenegro may i ask you what is the advantage of this step compared with setting the `CFBundleIdentifier` value?

**Comment by emenegro** [comment]
@godrei using `CFBundleIdentifier` is not recommended since Xcode 7.

From [Xcode 7 Release Notes](https://developer.apple.com/library/content/documentation/Xcode/Conceptual/RN-Xcode-Archive/Chapters/xc7_release_notes.html):
  > The new build setting Product Bundle Identifier (PRODUCT_BUNDLE_IDENTIFIER) is the recommended place to set the Bundle Identifier for a target. The target’s Info.plist should be configured to use this build setting by referencing it as $(PRODUCT_BUNDLE_IDENTIFIER) in the value for the CFBundleIdentifier key.

I commented this in the Q&A, [here](https://discuss.bitrise.io/t/ios-bundle-identifier-not-being-modified/1248/4).

**Comment by godrei** [comment]
@emenegro thanks for the infos, i did not realise that! 

This case i agree with the step!

Please create a new PR with only one commit. To do this please:

- close this PR
- remove the `set-ios-product-bundle-identifier-1.0.0` branch from your steplib fork
- share the step again

I'd suggest you to create a new version with the fixes, instead of re-sharing the 1.0.0 version. Simply fix + create 1.0.1 + share that.

Thanks for the PR!

**Pull request closed without merging**